### ISSUE TITLE
Interbnet-Explorer 9 Support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [josejaner/flot#441]
```
Hello!

I found the library doesn't work in Internet-Explorer 9. I debugged the 
problem, and solved it. It's very simple.

In jquery.flot.js in lines 539 and 553 (where browser is checked) add an 
additional condition to check whether an Internet Explorer browser has version 
9 or below.

As it was:
if ($.browser.msie)

As it should be:
if ($.browser.msie && parseInt($.browser.version) < 9)

It works fine for me.
```

Original issue reported on code.google.com by `evgeny.p...@gmail.com` on 1 Dec 2010 at 12:33
- Merged into: #408

**Comment by GoogleCodeExporter** [comment]
```
Thanks, it works fine for me too :)
```

Original comment by `matthieu...@gtempaccount.com` on 6 Dec 2010 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
Here is also a minified version 0.6 that include these changes.
```

Original comment by `evgeny.p...@gmail.com` on 6 Dec 2010 at 1:23

Attachments:
- [jquery.flot.min.js](https://storage.googleapis.com/google-code-attachments/flot/issue-441/comment-2/jquery.flot.min.js)

**Comment by GoogleCodeExporter** [comment]
Original comment by `olau%iol...@gtempaccount.com` on 13 Dec 2010 at 6:22
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
good stuff man...i just hit this issue
```

Original comment by `monemi...@gmail.com` on 17 May 2011 at 3:42

**Comment by GoogleCodeExporter** [comment]
```
I believe this was fixed in 0.7.
```

Original comment by `stevecmi...@gmail.com` on 25 Jul 2011 at 6:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `dnsch...@gmail.com` on 7 May 2012 at 11:29
